hal/devices/sierra_wireless/wp7702/Makefile

```diff
@@ -0,0 +1,2 @@
+wp7702.o: wp7702.c
```

Comment:
Let's unify filename. That is, `wp7702.c` should be changed into something like `entry.c` or `impl.c`